CMakeSettings.json

```diff
@@ -7,7 +7,7 @@
       "inheritEnvironments": [ "msvc_x64_x64" ],
       "buildRoot": "${projectDir}\\out\\build\\${name}",
       "installRoot": "${projectDir}\\out\\install\\${name}",
-      "cmakeCommandArgs": "-DBUILD_SAMPLES=true -DUSE_CPPRESTSDK=true",
+      "cmakeCommandArgs": "-DBUILD_SAMPLES=true -DUSE_CPPRESTSDK=true -DWERROR=true -DWALL=true",
```

Comment:
Is this needed when the CMakeLists already turns it on? Seems redundant unless I'm misreading the behavior.

Reply:
Ah true